frontend/src/utils.js

```diff
@@ -84,6 +86,32 @@ export const getHeaderText = location => {
   }
 };
 
+export const selectWebpageType = (
+  interceptExercise: string,
+  releaseCodes: string,
+  showWebpage: () => void,
+  actionType: string
+) => {
+  if (
+    actionType === 'email_attachment_download' ||
+    actionType === 'email_forwarded' ||
+    actionType === 'email_opened' ||
+    actionType === 'email_quick_reply' ||
+    actionType === 'email_replied'
```

Comment:
Put this into an array and check actiontype is in there

```
if (negativeActions.indexOf(actionType) > 0) { ... }
```
Can't use `includes` as we would need polyfill for IE. `indexOf` should be fine